strings/src/main/res/values/strings.xml

```diff
@@ -75,7 +75,7 @@
     <string name="storage_runtime_permission_denied_title">Storage permissions</string>
     <string name="storage_runtime_permission_denied_desc">Without these permissions Collect can\'t access your forms or save answers. Reopen the app when you are ready to grant them.</string>
     <string name="location_runtime_permissions_denied_title">Location permissions</string>
-    <string name="location_runtime_permissions_denied_desc">Without these permissions Collect can\'t record your location. Please try again when you are ready to grant them.</string>
+    <string name="location_runtime_permissions_denied_desc">Without <b>Precise Location</b> permission Collect can\'t record your location. Please try again when you are ready to grant it.</string>
```

Comment:
This would read better with a "the": "Without the Precise Location permission..."

(Pesky articles!)

Reply:
Right, fixed.